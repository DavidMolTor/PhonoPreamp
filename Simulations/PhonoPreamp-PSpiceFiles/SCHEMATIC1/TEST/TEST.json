{"TimeStamp":"Tue Aug  5 12:34:51 2025\n","Circuits":{"ExitCode":"0","TraceCount":"132","Mode":"SPICE","Type":"TIFullMode","Models":[{"ModelName":"X_U2.DX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U3.DX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U4.DX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U5.DX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U2.JX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"j","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U3.JX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"j","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U4.JX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"j","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U5.JX","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lf353.lib","ModelType":".model","DeviceType":"j","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.DBLOCK","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.DDEFAULT","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XD1.IDIODE","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XD2.IDIODE","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XD3.IDIODE","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XD4.IDIODE","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XU3.DVN","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"},{"ModelName":"X_U1.X1.XU4.DVNF","FileName":"c:\\cds_spb_home\\cdssetup\\psptilibdir\\lm4562.lib","ModelType":".model","DeviceType":"d","Source":"FROM_INC","isLibSigned":"true","isAllowed":"true"}]}}