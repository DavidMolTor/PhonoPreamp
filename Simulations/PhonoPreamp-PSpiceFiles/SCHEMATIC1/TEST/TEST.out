
**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "TEST.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\cds_spb_home\cdssetup\OrCAD_PSpiceTIPSpice_Install\23.1.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.AC DEC 100 20 20000
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source PHONOPREAMP
V_V1         VCC 0 12Vdc
V_V2         0 VEE 12Vdc
R_R1         0 N00420  150k TC=0,0 
C_C1         0 N00420  100p  TC=0,0 
R_R2         0 N00426  68.1k TC=0,0 
X_U1         N02071 N00426 VCC VEE N02631 LM4562 
R_R3         N006951 N02071  221 TC=0,0 
C_C2         0 N006951  220u  TC=0,0 
X_U2         N02071 N01030 VCC VEE N01098 LF353
R_R4         N01030 N009500  221 TC=0,0 
C_C3         N009500 0  220u  TC=0,0 
R_R5         N01098 N01030  4.22k TC=0,0 
R_R6         N01250 N01098  2k TC=0,0 
X_U3         0 N01250 VCC VEE N01384 LF353
R_R7         N01384 N01250  2k TC=0,0 
C_C4         N00420 N00426  1u  TC=0,0 
R_R8         N00426 N01384  1MEG TC=0,0 
R_R9         N02071 N02428  5.23k TC=0,0 
R_R10         N02428 N02631  63.4k TC=0,0 
C_C5         N02071 N02428  14.321n  TC=0,0 
C_C7         N02428 N02631  50n  TC=0,0 
C_C10         N02631 N03175  1u  TC=0,0 
C_C11         N03175 N03189  1u  TC=0,0 
C_C12         N03189 N03193  1u  TC=0,0 
R_R11         0 N03175  8.06k TC=0,0 
R_R12         0 N03193  31.6k TC=0,0 
R_R13         N03189 N03345  5.62k TC=0,0 
X_U4         N03193 N03345 VCC VEE N03345 LF353
C_C13         N03345 N033921  2.2u  TC=0,0 
R_R14         N033921 N03931  1.18k TC=0,0 
C_C14         0 N03931  2.2n  TC=0,0 
R_R15         0 N03931  23.2k TC=0,0 
X_U5         N03931 N04450 VCC VEE N04462 LF353
C_C15         N04450 N04462  100p  TC=0,0 
R_R16         N04450 N04549  24.9k TC=0,0 
C_C16         N04450 N04549  0.1u  TC=0,0 
X_R17         N045841 N04549 N04462 POT PARAMS: SET={GAIN} VALUE=3k
R_R18         N047341 N045841  1k TC=0,0 
C_C17         0 N047341  220u  TC=0,0 
R_R19         N06228 N04462  1k TC=0,0 
R_R20         0 N06228  100k TC=0,0 
V_V4         N00420 0 DC 0Vdc AC 0.1Vac 
X_U6         N11511 N11574 VCC VEE N11719 OPA161X
R_R21         0 N11064  47k TC=0,0 
C_C18         0 N11064  100p  TC=0,0 
C_C19         N11574 N11715  22n  TC=0,0 
R_R22         N11519 N11574  4.7k TC=0,0 
R_R23         N11064 N11511  2.2k TC=0,0 
C_C20         0 N11519  22u  TC=0,0 
R_R24         N11574 N11719  180k TC=0,0 
R_R25         N11715 N11719  10k TC=0,0 
X_U7         N13272 N11817 VCC VEE N12313 OPA161X
R_R26         N11942 N11817  2.7k TC=0,0 
C_C21         0 N11942  22u  TC=0,0 
R_R27         N11817 N12313  100k TC=0,0 
R_R28         N12313 N12291  750 TC=0,0 
C_C22         0 N12291  100n  TC=0,0 
R_R29         0 N12265  100k TC=0,0 
C_C23         N12291 N12265  1u  TC=0,0 
X_U8         N13350 N13140 VCC VEE N13140 LF353
C_C27         N13266 N13350  1u  TC=0,0 
R_R30         N13266 N13140  47.5k TC=0,0 
C_C24         N13140 N13272  1u  TC=0,0 
R_R33         0 N13350  90.9k TC=0,0 
R_R34         0 N13272  6.36k TC=0,0 
C_C26         N11719 N13266  1u  TC=0,0 
V_V5         N11064 0  
+SIN 0 5m 1k 0 0 0
.PARAM  gain=0.5

**** RESUMING TEST.cir ****
.END

INFO(ORPSIM-16573): AC analysis does not support expression evaluation at each frequency. ABM devices with non-linear operations will be evaluated only at bias point.

**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U2.DX         X_U3.DX         X_U4.DX         X_U5.DX         
          IS  800.000000E-18  800.000000E-18  800.000000E-18  800.000000E-18 


               X_U8.DX         X_U1.X1.DDEFAULT                X_U1.X1.DBLOCK  
          IS  800.000000E-18  800.000000E-18                  800.000000E-18 
          BV                   25            


               X_U6.X_U1.DVN   X_U6.X_U36.DVNF X_U6.X_U37.DVNF X_U7.X_U1.DVN   
          IS  100.000000E-18  100.000000E-18  100.000000E-18  100.000000E-18 
          KF    3.162278E-12    3.162278E-12    3.162278E-12    3.162278E-12 


               X_U7.X_U36.DVNF X_U7.X_U37.DVNF 
          IS  100.000000E-18  100.000000E-18 
          KF    3.162278E-12    3.162278E-12 


               X_U1.X1.XD4.IDIODE 
          IS   10.000000E-15 
           N     .01         
          BV   60            
         IBV    1.000000E-03 


               X_U1.X1.XD3.IDIODE 
          IS   10.000000E-15 
           N     .01         
          BV   60            
         IBV    1.000000E-03 


               X_U1.X1.XD2.IDIODE 
          IS   10.000000E-15 
           N     .01         
          BV   60            
         IBV    1.000000E-03 


               X_U1.X1.XD1.IDIODE 
          IS   10.000000E-15 
           N     .01         
          BV   60            
         IBV    1.000000E-03 


               X_U1.X1.XU4.DVNF                X_U1.X1.XU3.DVN 
          IS  100.000000E-18                  100.000000E-18 
          KF   31.622780E-12                   31.622780E-12 


**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     Junction FET MODEL PARAMETERS


******************************************************************************




               X_U2.JX         X_U3.JX         X_U4.JX         X_U5.JX         
               PJF             PJF             PJF             PJF             
         VTO   -1              -1              -1              -1            
        BETA  250.100000E-06  250.100000E-06  250.100000E-06  250.100000E-06 
          IS   12.500000E-12   12.500000E-12   12.500000E-12   12.500000E-12 


               X_U8.JX         
               PJF             
         VTO   -1            
        BETA  250.100000E-06 
          IS   12.500000E-12 


**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_U6.R_NOISELESS 
  T_Measured   27            
   T_Current -273.15         
           R    1            


               X_U7.R_NOISELESS 
  T_Measured   27            
   T_Current -273.15         
           R    1            


**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     Voltage Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U6.X_S1._S1   X_U6.X_S2._S2   
         RON     .01             .01         
        ROFF    1.000000E+12    1.000000E+12 
         VON     .01             .01         
        VOFF    0               0            


               X_U6.X_U20.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .7          
        VOFF    0            


               X_U6.X_U21.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .5          
        VOFF     .1          


               X_U6.X_U25.OL_SW 
         RON    1.000000E-03 
        ROFF    1.000000E+12 
         VON     .9          
        VOFF     .8          


               X_U6.X_U5.ESD_SW                X_U7.X_S1._S1   X_U7.X_S2._S2   
         RON   50                                .01             .01         
        ROFF    1.000000E+12                    1.000000E+12    1.000000E+12 
         VON     .5                              .01             .01         
        VOFF     .1                             0               0            


               X_U7.X_U20.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .7          
        VOFF    0            


               X_U7.X_U21.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .5          
        VOFF     .1          


               X_U7.X_U25.OL_SW 
         RON    1.000000E-03 
        ROFF    1.000000E+12 
         VON     .9          
        VOFF     .8          


               X_U7.X_U5.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .5          
        VOFF     .1          


**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     SMALL SIGNAL BIAS SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  VCC)   12.0000  (  VEE)  -12.0000  (N00420)    0.0000 (N00426)-934.5E-06     

(N01030)-826.6E-06 (N01098)-826.5E-06 (N01250) 1.899E-06 (N01384) 830.3E-06     

(N02071)-828.5E-06 (N02428)-804.8E-06 (N02631)-516.3E-06 (N03175)    0.0000     

(N03189) 737.5E-09 (N03193)-1.170E-06 (N03345) 737.5E-09 (N03931)-859.1E-09     

(N04450) 1.048E-06 (N04462) 2.026E-06 (N04549) 1.971E-06 (N06228) 2.006E-06     

(N11064)    0.0000 (N11511)-132.0E-06 (N11519)-32.54E-06 (N11574)-32.54E-06     

(N11715)     .0063 (N11719)     .0063 (N11817)-282.1E-06 (N11942)-282.1E-06     

(N12265)    0.0000 (N12291)     .0032 (N12313)     .0032 (N13140)-1.458E-06     

(N13266)-1.458E-06 (N13272)-381.6E-06 (N13350)-3.366E-06 (X_U2.6) 259.4E-09     

(X_U2.7)-847.2E-06 (X_U2.8)-847.2E-06 (X_U2.9)    0.0000 (X_U3.6)-260.7E-09     

(X_U3.7) 851.1E-06 (X_U3.8) 851.1E-06 (X_U3.9)    0.0000 (X_U4.6)-225.3E-12     

(X_U4.7) 739.3E-09 (X_U4.8) 739.3E-09 (X_U4.9)    0.0000 (X_U5.6)-617.9E-12     

(X_U5.7) 2.029E-06 (X_U5.8) 2.029E-06 (X_U5.9)    0.0000 (X_U8.6) 443.2E-12     

(X_U8.7)-1.457E-06 (X_U8.8)-1.457E-06 (X_U8.9)    0.0000 (N006951)-828.5E-06    

(N009500)-826.6E-06                   (N033921)-859.1E-09                       

(N045841) 1.971E-06                   (N047341) 1.971E-06                       

(X_U2.10)    -.3759                   (X_U2.11)  -11.6550                       

(X_U2.12)  -11.6550                   (X_U2.53)    9.8000                       

(X_U2.54)   -9.8000                   (X_U2.90)-414.1E-06                       

(X_U2.91)   30.0000                   (X_U2.92)  -30.0000                       

(X_U2.99)    0.0000                   (X_U3.10)    -.3750                       

(X_U3.11)  -11.6550                   (X_U3.12)  -11.6550                       

(X_U3.53)    9.8000                   (X_U3.54)   -9.8000                       

(X_U3.90) 416.0E-06                   (X_U3.91)   30.0000                       

(X_U3.92)  -30.0000                   (X_U3.99)    0.0000                       

(X_U4.10)    -.3750                   (X_U4.11)  -11.6550                       

(X_U4.12)  -11.6550                   (X_U4.53)    9.8000                       

(X_U4.54)   -9.8000                   (X_U4.90) 37.03E-09                       

(X_U4.91)   30.0000                   (X_U4.92)  -30.0000                       

(X_U4.99)    0.0000                   (X_U5.10)    -.3750                       

(X_U5.11)  -11.6550                   (X_U5.12)  -11.6550                       

(X_U5.53)    9.8000                   (X_U5.54)   -9.8000                       

(X_U5.90) 57.09E-09                   (X_U5.91)   30.0000                       

(X_U5.92)  -30.0000                   (X_U5.99)    0.0000                       

(X_U8.10)    -.3750                   (X_U8.11)  -11.6550                       

(X_U8.12)  -11.6550                   (X_U8.53)    9.8000                       

(X_U8.54)   -9.8000                   (X_U8.90) 37.03E-09                       

(X_U8.91)   30.0000                   (X_U8.92)  -30.0000                       

(X_U8.99)    0.0000                   (X_U6.MID)    0.0000                      

(X_U6.V11)-13.79E-09                  (X_U6.V12)-13.71E-09                      

(X_U7.MID)    0.0000                  (X_U7.V11)-13.79E-09                      

(X_U7.V12)-13.71E-09                  (X_U1.X1.5)-804.5E-06                     

(X_U1.X1.6)-804.5E-06                 (X_U1.X1.8)-516.3E-06                     

(X_U1.X1.9)-516.3E-06                 (X_U6.ESDN)-32.54E-06                     

(X_U6.ESDP)-132.0E-06                 (X_U6.VCLP)     .0070                     

(X_U7.ESDN)-282.1E-06                 (X_U7.ESDP)-381.6E-06                     

(X_U7.VCLP)     .0036                 (X_U1.X1.10)-516.3E-06                    

(X_U1.X1.14)-516.3E-06                (X_U1.X1.15)-804.5E-06                    

(X_U1.X1.16)-804.5E-06                (X_U1.X1.17)-934.5E-06                    

(X_U1.X1.18)-516.3E-06                (X_U1.X1.19)-804.5E-06                    

(X_U1.X1.20)-828.5E-06                (X_U1.X1.21)-934.5E-06                    

(X_U6.CLAMP)     .0070                (X_U6.VCC_B)   12.0000                    

(X_U6.VEE_B)  -12.0000                (X_U6.VIMON) 35.00E-06                    

(X_U7.CLAMP)     .0036                (X_U7.VCC_B)   12.0000                    

(X_U7.VEE_B)  -12.0000                (X_U7.VIMON) 35.00E-06                    

(X_U6.N24207)-132.0E-06               (X_U6.N24458)-21.47E-09                   

(X_U6.N24659)-152.8E-12               (X_U6.N25248)-152.8E-12                   

(X_U6.N25397)-34.71E-06               (X_U6.N25415)-34.71E-06                   

(X_U6.N25921)   -1.3886               (X_U6.N25929)-1.102E-06                   

(X_U6.N27785)    1.4634               (X_U6.N27793) 1.084E-06                   

(X_U6.N28438)-34.71E-06               (X_U6.N28678)-32.52E-06                   

(X_U6.N31303)-32.54E-06               (X_U6.N31389)   10.0000                   

(X_U6.N31737)  -10.0000               (X_U6.N32236)-32.52E-06                   

(X_U6.N32347)-32.52E-06               (X_U6.N32510) 1.329E-06                   

(X_U6.N34005)     .1517               (X_U6.N34013)     .0070                   

(X_U6.N36262)     .0070               (X_U6.N37029)     .0694                   

(X_U6.N37037) 256.5E-06               (X_U6.N39124)     .0694                   

(X_U6.N39417)     .0694               (X_U6.N39473)     .0694                   

(X_U6.N39884)     .0694               (X_U6.N39892) 64.30E-06                   

(X_U6.N45999)     .0063               (X_U6.N46041)     .0694                   

(X_U6.N55560)    0.0000               (X_U6.N56849)    0.0000                   

(X_U6.N57321)    0.0000               (X_U6.N57325)    0.0000                   

(X_U6.N59739)   49.8000               (X_U6.N59743)    0.0000                   

(X_U6.N59747)    0.0000               (X_U6.N59865)    0.0000                   

(X_U6.N59881)    0.0000               (X_U6.N59905)  -47.0000                   

(X_U6.N65567)   48.0000               (X_U6.N65575)    0.0000                   

(X_U6.N65579)    0.0000               (X_U6.N65697)    0.0000                   

(X_U6.N65713)    0.0000               (X_U6.N65737)  -47.0000                   

(X_U6.N68774)     .0070               (X_U6.N72064)   11.7870                   

(X_U6.N72073)  -11.7870               (X_U6.N72391)  -11.7870                   

(X_U6.N72398)   11.7870               (X_U6.N81574)-13.79E-09                   

(X_U6.N81664)-13.71E-09               (X_U6.N81756)    0.0000                   

(X_U6.VOUT_S)     .0063               (X_U6.VSENSE)     .0070                   

(X_U6.X_U1.3)    0.0000               (X_U6.X_U1.4)    0.0000                   

(X_U6.X_U1.5)    0.0000               (X_U6.X_U1.6)    0.0000                   

(X_U6.X_U1.7)     .8338               (X_U6.X_U1.8)     .8338                   

(X_U7.N24207)-381.6E-06               (X_U7.N24458)-62.07E-09                   

(X_U7.N24659)-441.7E-12               (X_U7.N25248)-441.7E-12                   

(X_U7.N25397)-284.3E-06               (X_U7.N25415)-284.3E-06                   

(X_U7.N25921)   -1.3886               (X_U7.N25929)-1.102E-06                   

(X_U7.N27785)    1.4634               (X_U7.N27793) 1.084E-06                   

(X_U7.N28438)-284.3E-06               (X_U7.N28678)-282.1E-06                   

(X_U7.N31303)-282.1E-06               (X_U7.N31389)   10.0000                   

(X_U7.N31737)  -10.0000               (X_U7.N32236)-282.1E-06                   

(X_U7.N32347)-282.1E-06               (X_U7.N32510) 690.0E-09                   

(X_U7.N34005)     .0780               (X_U7.N34013)     .0036                   

(X_U7.N36262)     .0036               (X_U7.N37029)     .0359                   

(X_U7.N37037) 132.6E-06               (X_U7.N39124)     .0359                   

(X_U7.N39417)     .0359               (X_U7.N39473)     .0359                   

(X_U7.N39884)     .0359               (X_U7.N39892) 33.24E-06                   

(X_U7.N45999)     .0032               (X_U7.N46041)     .0359                   

(X_U7.N55560)    0.0000               (X_U7.N56849)    0.0000                   

(X_U7.N57321)    0.0000               (X_U7.N57325)    0.0000                   

(X_U7.N59739)   49.8000               (X_U7.N59743)    0.0000                   

(X_U7.N59747)    0.0000               (X_U7.N59865)    0.0000                   

(X_U7.N59881)    0.0000               (X_U7.N59905)  -47.0000                   

(X_U7.N65567)   48.0000               (X_U7.N65575)    0.0000                   

(X_U7.N65579)    0.0000               (X_U7.N65697)    0.0000                   

(X_U7.N65713)    0.0000               (X_U7.N65737)  -47.0000                   

(X_U7.N68774)     .0036               (X_U7.N72064)   11.7870                   

(X_U7.N72073)  -11.7870               (X_U7.N72391)  -11.7870                   

(X_U7.N72398)   11.7870               (X_U7.N81574)-13.79E-09                   

(X_U7.N81664)-13.71E-09               (X_U7.N81756)    0.0000                   

(X_U7.VOUT_S)     .0032               (X_U7.VSENSE)     .0036                   

(X_U7.X_U1.3)    0.0000               (X_U7.X_U1.4)    0.0000                   

(X_U7.X_U1.5)    0.0000               (X_U7.X_U1.6)    0.0000                   

(X_U7.X_U1.7)     .8338               (X_U7.X_U1.8)     .8338                   

(X_U1.X1.GNDF)    0.0000              (X_U6.N158495)     .0070                  

(X_U6.N161737) 35.00E-06              (X_U6.N161937)     .0063                  

(X_U6.VCC_CLP)   11.7870              (X_U6.VEE_CLP)  -11.7870                  

(X_U6.X_U36.3)    0.0000              (X_U6.X_U36.4)    0.0000                  

(X_U6.X_U36.5)    0.0000              (X_U6.X_U36.6)    0.0000                  

(X_U6.X_U36.7)     .8338              (X_U6.X_U36.8)     .8338                  

(X_U6.X_U37.3)    0.0000              (X_U6.X_U37.4)    0.0000                  

(X_U6.X_U37.5)    0.0000              (X_U6.X_U37.6)    0.0000                  

(X_U6.X_U37.7)     .8338              (X_U6.X_U37.8)     .8338                  

(X_U7.N158495)     .0036              (X_U7.N161737) 35.00E-06                  

(X_U7.N161937)     .0032              (X_U7.VCC_CLP)   11.7870                  

(X_U7.VEE_CLP)  -11.7870              (X_U7.X_U36.3)    0.0000                  

(X_U7.X_U36.4)    0.0000              (X_U7.X_U36.5)    0.0000                  

(X_U7.X_U36.6)    0.0000              (X_U7.X_U36.7)     .8338                  

(X_U7.X_U36.8)     .8338              (X_U7.X_U37.3)    0.0000                  

(X_U7.X_U37.4)    0.0000              (X_U7.X_U37.5)    0.0000                  

(X_U7.X_U37.6)    0.0000              (X_U7.X_U37.7)     .8338                  

(X_U7.X_U37.8)     .8338              (X_U1.X1.Vimon) 4.549E-09                 

(X_U1.X1.XU3.3)    0.0000             (X_U1.X1.XU3.4)    0.0000                 

(X_U1.X1.XU3.5)    0.0000             (X_U1.X1.XU3.6)    0.0000                 

(X_U1.X1.XU3.7)     .8338             (X_U1.X1.XU3.8)     .8338                 

(X_U1.X1.XU4.3)    0.0000             (X_U1.X1.XU4.4)    0.0000                 

(X_U1.X1.XU4.5)    0.0000             (X_U1.X1.XU4.6)    0.0000                 

(X_U1.X1.XU4.7)     .8338             (X_U1.X1.XU4.8)     .8338                 

(X_U1.X1.XU7.1) 24.00E-06             (X_U1.X1.XU7.2)    0.0000                 

(X_U1.X1.XU8.1)-254.4E-12             (X_U1.X1.XU8.2)    0.0000                 

(X_U6.CL_CLAMP)     .0070             (X_U6.N1356061)  -13.7520                 

(X_U6.N1356115)   13.6860             (X_U6.N1356333)  -13.7520                 

(X_U6.N1356345)  -13.7520             (X_U6.N1356409)   13.6860                 

(X_U6.N1356429)   13.6860             (X_U6.N2392113)    0.0000                 

(X_U6.N2392253)   12.0000             (X_U6.N2392293)  -12.0000                 

(X_U6.N2402147)-152.8E-12             (X_U6.N2402155)-1.087E-12                 

(X_U6.N2419074)     .0070             (X_U6.N2423000)     .1517                 

(X_U7.CL_CLAMP)     .0036             (X_U7.N1356061)  -13.7550                 

(X_U7.N1356115)   13.6830             (X_U7.N1356333)  -13.7550                 

(X_U7.N1356345)  -13.7550             (X_U7.N1356409)   13.6830                 

(X_U7.N1356429)   13.6830             (X_U7.N2392113)    0.0000                 

(X_U7.N2392253)   12.0000             (X_U7.N2392293)  -12.0000                 

(X_U7.N2402147)-441.7E-12             (X_U7.N2402155)-3.144E-12                 

(X_U7.N2419074)     .0036             (X_U7.N2423000)     .0779                 

(X_U1.X1.NET126)   11.5980            (X_U1.X1.NET128)  -25.0980                

(X_U6.CLAW_CLAMP)     .0070           (X_U7.CLAW_CLAMP)     .0036               

(X_U1.X1.XU11.VP1)-516.3E-06          (X_U1.X1.XU11.VP2)-516.3E-06              

(X_U1.X1.XU11.VP3)-516.3E-06          (X_U1.X1.XU11.VP4)-516.3E-06              

(X_U1.X1.XU11.VX1)    0.0000          (X_U1.X1.XU11.VX2)    0.0000              

(X_U1.X1.XU11.VX3)    0.0000          (X_U1.X1.XU11.VX4) 108.4E-21              

(X_U1.X1.XU11.VX5)-108.4E-21          (X_U1.X1.XU11.VZ1)-516.3E-06              

(X_U1.X1.XU11.VZ2)-516.3E-06          (X_U1.X1.XU11.VZ3)-516.3E-06              

(X_U1.X1.XU11.VZ4)-516.3E-06          (X_U6.SW_OL_OPA161x)    0.0000            

(X_U7.SW_OL_OPA161x)    0.0000        (X_U1.X1.XU5.VDD_CLP)   11.0000           

(X_U1.X1.XU5.VSS_CLP)  -11.0000       




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1        -2.118E-02
    V_V2        -2.118E-02
    V_V4         0.000E+00
    V_V5        -6.000E-08
    X_U2.VB      2.594E-12
    X_U2.VC      9.802E-12
    X_U2.VE      9.800E-12
    X_U2.VLIM   -4.141E-07
    X_U2.VLP    -3.000E-11
    X_U2.VLN    -3.000E-11
    X_U3.VB     -2.607E-12
    X_U3.VC      9.800E-12
    X_U3.VE      9.802E-12
    X_U3.VLIM    4.160E-07
    X_U3.VLP    -3.000E-11
    X_U3.VLN    -3.000E-11
    X_U4.VB     -2.252E-15
    X_U4.VC      9.801E-12
    X_U4.VE      9.801E-12
    X_U4.VLIM    3.703E-11
    X_U4.VLP    -3.000E-11
    X_U4.VLN    -3.000E-11
    X_U5.VB     -6.178E-15
    X_U5.VC      9.801E-12
    X_U5.VE      9.801E-12
    X_U5.VLIM    5.709E-11
    X_U5.VLP    -3.000E-11
    X_U5.VLN    -3.000E-11
    X_U6.V_VCM_MAX  -1.000E-11
    X_U6.V_VCM_MIN   1.000E-11
    X_U6.V_V_GRN   4.700E-11
    X_U6.V_V_GRP  -4.800E-11
    X_U6.V_V_ISCN   4.700E-11
    X_U6.V_V_ISCP  -4.980E-11
    X_U6.V_V_ORN   0.000E+00
    X_U6.V_V_ORP   0.000E+00
    X_U7.V_VCM_MAX  -1.000E-11
    X_U7.V_VCM_MIN   1.000E-11
    X_U7.V_V_GRN   4.700E-11
    X_U7.V_V_GRP  -4.800E-11
    X_U7.V_V_ISCN   4.700E-11
    X_U7.V_V_ISCP  -4.980E-11
    X_U7.V_V_ORN   0.000E+00
    X_U7.V_V_ORP   0.000E+00
    X_U8.VB      4.432E-15
    X_U8.VC      9.801E-12
    X_U8.VE      9.801E-12
    X_U8.VLIM    3.703E-11
    X_U8.VLP    -3.000E-11
    X_U8.VLN    -3.000E-11
    X_U1.X1.Vos  -1.549E-08
    X_U6.X_H1.VH_H1   3.500E-08
    X_U6.X_H2.VH_H2  -1.371E-08
    X_U6.X_H3.VH_H3   1.379E-08
    X_U7.X_H1.VH_H1   3.500E-08
    X_U7.X_H2.VH_H2  -1.371E-08
    X_U7.X_H3.VH_H3   1.379E-08
    X_U1.X1.XU6.VSENSE   4.549E-09

    TOTAL POWER DISSIPATION   5.08E-01  WATTS



          JOB CONCLUDED

**** 08/08/25 17:34:45 **** PSpice 23.1.0 (30 January 2024) *** ID# 0 ********

 ** Profile: "SCHEMATIC1-TEST"  [ d:\otros\electrónica\phonopreamp\simulations\phonopreamp-pspicefiles\schematic1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        1.94
